content/300-guides/025-migrate/500-data-migration.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: Data migrations
+metaDescription: How to migrate data using Prisma using the expand and contract pattern.
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma does not yet natively support data migrations, but you can use the [expand and contract pattern](https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern) to migrate your data. For example from one column into another.
+
+This guide covers how you can use Prisma with the expand and contract pattern to:
+
+- Expand your schema with a new column
+- Create and run the data migration
+- Contract your schema by dropping the old column
+
+</TopBlock>
+
+## Overview of the steps
+
+This tutorial will walk you through the following steps:
+
+1. Expand your schema with a new column
+1. Create and run the data migration file
+1. Contract your schema by dropping the old column
+
+It also makes the following assumptions:
+
+- The production database is accessible from the development machine
+- `prisma migrate dev` is only run against development database 
+- The expanding and contracting steps are handled in separate branches
+
+For this guide, you will modify the following schema by replacing the `published` boolean field with a `status` enum:
+
+```prisma file=prisma/schema.prisma
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  title     String
+  content   String?
+  published Boolean  @default(false)
+  viewCount Int      @default(0)
+}
+```
+
+## Expand your schema with a new column
+
+Checkout to a new branch from your `main` branch:
+
+```bash
+git checkout -b create-status-field
+```
+
+Make the following updates to your Prisma schema:
+
+- Create a `Status` enum with the following values: `Unknown`, `Draft`, `InReview`, and `Published`
+- Add a `status` column to the `Post` model
+- Mark the `published` field as optional
+
+```prisma file=prisma/schema.prisma highlight=12,13,18-24;edit
```

Comment:
Is this highlighting correct? 

shouldn't it instead:
- edit line 6
- add lines 9-15

<img width="697" alt="image" src="https://github.com/prisma/docs/assets/4058327/6f075a7a-5180-46ae-ba7e-bb7d97897fdd">